src/password/landing.js

```diff
@@ -1,17 +1,22 @@
 document.addEventListener('DOMContentLoaded', () => {
 if(localStorage.getItem('storedHashedPassword')){
+    document.getElementById('welcome-button').style.display = 'none'
 //Wait for input password to be submitted
 document.getElementById('input-password-form').addEventListener('submit', async function(event){
+    console.log("here")
     event.preventDefault();
     const inputPassword = document.getElementById('password').value;
+    //hashed version of the input
     const hashedInputPassword = await hashPassword(inputPassword);
+    //retrive the stored hashed password
     const storedHashedPassword = localStorage.getItem('storedHashedPassword');
-    //check if input password is the same as the stored password
+    //set the authentication key to the hashed input password. this will be hashed again to verify the user has logged in
+    sessionStorage.setItem('authenticateKey', hashedInputPassword)
+    //check if input password is the same as the stored password, and redirect to the calendar if they are
     if(hashedInputPassword === storedHashedPassword){
         window.location.href = '/src/calendar/calendar.html';
```

Comment:
change to be a relative link, e.g. '../calendar/calendar.html' so that cloudflare doesn't break (it's hosted on src so it can't see /src)